.github/workflows/integrationtests.yaml

```diff
@@ -127,4 +127,6 @@ jobs:
       - name: Download opentelemetry javaagent.
         run: wget -q https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.0/opentelemetry-javaagent.jar
       - name: Micrometer Integration Tests.
-        run: ./gradlew -p core/micrometer test -DincludeTags="OpenTelemetryIntegrationTest"
+        run: ./gradlew :core:micrometer:test -DincludeTags="OpenTelemetryIntegrationTest"
+      - name: Tracing Integration Tests.
+        run: ./gradlew :system-tests:tests:test -DincludeTags="OpenTelemetryIntegrationTest"
```

Comment:
nit: any reason we do not use -p to target here modules as we do in other steps ? Because maybe this works now but if we another submodule inside then it might not be able to execute all tests.

And also for consistency.

Reply:
I did it this way because it was not needed, but sure I can change it for consistency